src/generators/essudpgen/ReadoutGeneratorBase.cpp

```diff
@@ -11,13 +11,16 @@
 #include <cassert>
 #include <common/debug/Trace.h>
 #include <generators/essudpgen/ReadoutGeneratorBase.h>
+#include <iostream>
+#include <iterator>
```

Comment:
I can't see where you are using these two headers. Did I miss somehting?
If they are not used here they should probably be moved to another .h or .cpp
file.

Reply:
Got spot, I cleared the includes, actually there were a bit more which were unecessary.